src/SwedbankPay.Sdk/Payments/SwishPayments/SwishPaymentOperations.cs

```diff
@@ -0,0 +1,51 @@
+﻿using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.SwishPayments
+{
+    public class SwishPaymentOperations : OperationsBase
+    {
+        public SwishPaymentOperations(OperationList operations, HttpClient client)
+        {
+            foreach (var httpOperation in operations)
+            {
+                switch (httpOperation.Rel.Value)
+                {
+                    case PaymentResourceOperations.UpdatePaymentAbort:
+                        Abort = async () =>
+                            await client.SendAsJsonAsync<SwishPaymentPaymentResponse>(httpOperation.Method, httpOperation.Href, new PaymentAbortRequest());
```

Comment:
Shouldn't `PaymentAbortRequest` be passed in as a `payload` argument?

```suggestion
                            await client.SendAsJsonAsync<SwishPaymentPaymentResponse>(httpOperation.Method, httpOperation.Href, payload);
```